python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -0,0 +1,250 @@
+
+from Ganga.GPIDev.Adapters.IPrepareApp import IPrepareApp
+from Ganga.GPIDev.Schema import Schema, Version, SimpleItem, GangaFileItem
+
+from Ganga.GPIDev.Lib.File.File import ShareDir
+from Ganga.Core import ApplicationConfigurationError, GangaException
+
+from Ganga.Utility.logging import getLogger
+
+from Ganga.GPIDev.Base.Proxy import getName
+
+from os import rename, path, makedirs, chdir
```

Comment:
Yes but we don't follow this across the codebase atm. It would be more efficient to try and do this automatically.